src/commands/moderation/kick.ts

```diff
@@ -38,50 +38,64 @@ export default class Kick {
       ].join("\n"))
       .setThumbnail(user.displayAvatarURL())
-      .setAuthor({ name: `• ${user.username}`, iconURL: user.displayAvatarURL() })
       .setFooter({ text: `User ID: ${user.id}` })
       .setColor(genColor(100));
 
     const embedDM = new EmbedBuilder()
+      .setAuthor({ name: `• ${user.username}`, iconURL: user.displayAvatarURL() })
       .setTitle(`👢 • You were kicked`)
       .setDescription([
         `**Moderator**: ${interaction.user.username}`,
         `**Reason**: ${reason ?? "No reason provided"}`
       ].join("\n"))
       .setThumbnail(user.displayAvatarURL())
-      .setAuthor({ name: `• ${user.username}`, iconURL: user.displayAvatarURL() })
       .setFooter({ text: `User ID: ${user.id}` })
       .setColor(genColor(0));
 
-    if (!member.permissions.has(PermissionsBitField.Flags.KickMembers))
-      return await interaction.followUp({ embeds: [errorEmbed("You need the **Kick Members** permission to execute this command.")] });
-    if (selectedMember === member)
-      return await interaction.followUp({ embeds: [errorEmbed("You can't kick yourself.")] });
-    if (selectedMember.user.id === interaction.client.user.id)
-      return await interaction.followUp({ embeds: [errorEmbed("You can't kick Nebula.")] });
-    if (!selectedMember.manageable)
-      return await interaction.followUp({ embeds: [errorEmbed(`You can't kick ${name}, because they have a higher role position than Nebula.`)] });
-    if (member.roles.highest.position < selectedMember.roles.highest.position)
-      return await interaction.followUp({ embeds: [errorEmbed(`You can't kick ${name}, because they have a higher role position than you.`)] });
+    if (!member.permissions.has(PermissionsBitField.Flags.KickMembers)) return await interaction.followUp({
+      embeds: [errorEmbed("You need the **Kick Members** permission to execute this command.")]
+    });
+    
+    if (selectedMember === member) return await interaction.followUp({
+      embeds: [errorEmbed("You can't kick yourself.")]
+    });
+    
+    if (selectedMember.user.id === interaction.client.user.id) return await interaction.followUp({
+      embeds: [errorEmbed("You can't kick Nebula.")]
+    });
+    
+    if (!selectedMember.manageable) return await interaction.followUp({
+      embeds: [errorEmbed(`You can't kick ${name}, because they have a higher role position than Nebula.`)]
+    });
+    
+    if (member.roles.highest.position < selectedMember.roles.highest.position) return await interaction.followUp({
+      embeds: [errorEmbed(`You can't kick ${name}, because they have a higher role position than you.`)]
+    });
 
     const db = this.db;
     const settingsTable = await getSettingsTable(db);
-    const logChannel = await settingsTable?.get(`${interaction.guild.id}.logChannel`).then(
-      (channel: string | null) => channel ?? null
-    ).catch(() => null);
+    const logChannel = await settingsTable
+      ?.get(`${interaction.guild.id}.logChannel`)
```

Comment:
```suggestion
    const logChannel = await settingsTable?
      .get(`${interaction.guild.id}.logChannel`)
```
That character belongs there I think ;)